src/Skoruba.IdentityServer4.Admin.BusinessLogic.Identity/Skoruba.IdentityServer4.Admin.BusinessLogic.Identity.csproj

```diff
@@ -2,7 +2,7 @@
 
 	<PropertyGroup>
```

Comment:
Maybe we could manage these common properties in a `Directory.Build.props`, it'll be used for all the projects under this folder by MS Build implicitly,  I'd like to provide a PR if needed.

<https://docs.microsoft.com/en-us/visualstudio/msbuild/customize-your-build?view=vs-2019#directorybuildprops-and-directorybuildtargets>